docs/admin/networking/index.md

```diff
@@ -77,6 +77,64 @@ direct.
 In general, [port forwarded](./port-forwarding.md) web apps are faster than
 dashboard-accessed web apps.
 
+## Latency
+
+Coder measures and reports several types of latency, providing insights into the performance of your deployment. Understanding these metrics can help you diagnose issues and optimize the user experience.
+
+There are three main types of latency metrics for your Coder deployment:
+
+- Dashboard-to-server latency:
+  
+  The Coder UI measures round-trip time to the Coder server using the browser's Performance API.
+  
+  This appears in the user interface next to your username, showing how responsive the dashboard is.
+
+- Workspace connection latency:
+  
+  When users connect to workspaces, Coder measures and displays the latency between the user and:
+  - Workspace (direct P2P connection when possible)
+  - DERP relay servers (when P2P isn't possible)
+
+  This latency is visible in workspace cards and resource pages. It shows the round-trip time in milliseconds.
```

Comment:
Ok, I've looked into this, and admittedly I'm a bit embarassed by what we built.

The latency on the Agent resource card is between the Agent and the shown relay. It never shows P2P (which isn't possible to achieve via the web browser anyway).

The estimated total latency if you were to access a workspace via the browser would be if you added the numbers by your username to the numbers you see in the Agent card, then that would give you a rough picture of the total latency thru various relays.